controller/src/main/java/io/pravega/controller/store/stream/PravegaTableScope.java

```diff
@@ -39,7 +39,7 @@
  * Each such scope table is protected against recreation of scope by attaching a unique id to the scope when it is created. 
  */
 public class PravegaTableScope implements Scope {
```

Comment:
Can you opportunistically fix the name here to `PravegaTablesScope` to be consistent with the naming of other classes about pravda tables?

Reply:
done